btk/draw_blends.py

```diff
@@ -41,28 +42,6 @@ def get_center_in_pixels(blend_catalog, wcs):
     return dx_col, dy_col
 
 
-def get_size(catalog, psf, pixel_scale):
```

Comment:
Did you remove this because you think it was unnecessary later on? I agree if that's the case

Reply:
Yes.